src/ui_fsmenu/addons/remote_interaction.cc

```diff
@@ -370,6 +369,108 @@ void CommentEditor::reset_text() {
 	think();
 }
 
+/* TransifexSettingsBox implementation */
+
+class TransifexSettingsBox : public UI::Box {
+public:
+	TransifexSettingsBox(UI::Box& parent, std::shared_ptr<AddOns::AddOnInfo> info)
+	   : UI::Box(&parent, UI::PanelStyle::kFsMenu, 0, 0, UI::Box::Vertical),
+	     priority_(this, 0, 0, 0, 0, UI::PanelStyle::kFsMenu),
+	     name_(this, 0, 0, 450, UI::PanelStyle::kFsMenu),
+	     categories_(this, 0, 0, 0, UI::PanelStyle::kFsMenu) {
+		add(new UI::Textarea(this, UI::PanelStyle::kFsMenu, UI::FontStyle::kFsMenuInfoPanelHeading,
+		                     pgettext("tx", "Priority:"), UI::Align::kCenter),
+		    UI::Box::Resizing::kFullSize);
+		add_space(kRowButtonSpacing);
+		add(&priority_, UI::Box::Resizing::kFullSize);
+		add_space(kRowButtonSpacing);
+		add(new UI::Textarea(
+		       this, UI::PanelStyle::kFsMenu, UI::FontStyle::kFsMenuInfoPanelHeading,
+		       /** TRANSLATORS: "Resource" here refers to the name of a translation unit */
+		       pgettext("tx", "Resource Name:"), UI::Align::kCenter),
```

Comment:
This refers to the name which is displayed to the translators. This is usually the add-on's name but not it can also be set to something different:
![grafik](https://user-images.githubusercontent.com/48293446/142856527-9233e60d-1f16-4b7f-bd4e-2192e38ded7a.png)
https://github.com/widelands/wl_addons_server/blob/df10f53b49a9f5b3c8d6e6605e6323eecad30cbe/wl/server/Command.java#L473

Reply:
Oh, I mistook it with the parameter of the command.